python/cudf/cudf/core/indexed_frame.py

```diff
@@ -699,6 +699,37 @@ def drop_duplicates(
             self._index.names if not ignore_index else None,
         )
 
+    @annotate("FRAME_EMPTY_LIKE", color="green", domain="cudf_python")
+    def _empty_like(self, keep_index=True):
+        return self._from_columns_like_self(
+            libcudf.copying.columns_empty_like(
+                [
+                    *(self._index._data.columns if keep_index else ()),
+                    *self._columns,
+                ]
+            ),
+            self._column_names,
+            self._index.names if keep_index else None,
+        )
+
+    def _split(self, splits, keep_index=True):
+        columns_splitted = libcudf.copying.columns_split(
```

Comment:
Past tense of split is split.
```suggestion
        columns_split = libcudf.copying.columns_split(
```